### PR TITLE
Improved definitions for awesome-notifications

### PR DESCRIPTION
**Pull request by seebeen**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<[url here](https://github.com/f3oall/awesome-notifications/issues/57)>>

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/awesome-notifications/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against df7bc952c8963393eb8316f72db076f59217a05b
</p>

**Comment by seebeen** [comment]
I mistakenly thought that adding an org would allow all of our org members to approve PRs here. I added myself back :)

File: types/awesome-notifications/index.d.ts [hunk 1]
```diff
@@ -1,103 +1,155 @@
...
+            modal?: string;
+            general?: {
+                "<script>"?: string;
+                "</script>"?: string;
+            };
+        };
+
+    interface AwnOptions<E = Error> {
+        /**
+         * Defines position of the toast. Can be ‘bottom-right’, ‘bottom-left’, ‘top-left’, ‘top-right’.
```
Comment (codershiba) [inline types/awesome-notifications/index.d.ts:53]:
```suggestion
         * Defines position of the toast. Can be 'bottom-right', 'bottom-left', 'top-left', 'top-right'.
```

File: types/awesome-notifications/index.d.ts [hunk 2]
```diff
@@ -219,8 +241,14 @@ export default class AWN {
      * @param  onCancel Defines Function, which will be executed on click to ‘Cancel’ button.
      *                  If false was passed, ‘Cancel’ button will be hidden.
      * @param  options  Instance of `AwnOptions`, which will override globals for this call
-     * @return          A new `HTMLElement` instance
+     * @return          A new `HTMLDivElement` instance
```
Comment (codershiba) [inline types/awesome-notifications/index.d.ts:244]:
```suggestion
     * @return          A new HTMLDivElement instance
```

Comment (seebeen) [reply types/awesome-notifications/index.d.ts]: Recommendation implemented

File: types/awesome-notifications/index.d.ts (hunk 1 above)
Comment (seebeen) [reply types/awesome-notifications/index.d.ts]: Recommendation implemented

**Review by ghost — APPROVED** [review]

**Comment by seebeen** [comment]
Ready to merge

**Pull request merged**